docs/cluster_management.md

```diff
@@ -2,45 +2,152 @@
+
+## Prerequisites
+
+Kubernetes is written in Go and with some work it should be possible to run it anywhere where Docker can run (yes, this includes Raspberry Pi). Due to differences between Linux distributions and between cloud providers setting it up requires different amount of work on different environments. The best and simplest experience is provided by using Google Container Engine [GKE](https://cloud.google.com/container-engine/) or Google Compute Engine [GCE](https://cloud.google.com/compute/). Other environments are supported and maintained by the community, Vagrant or CoreOS on various IaaS providers being two examples.
+
+To start using Kubernetes you need an access to machine(s) of one of supported configurations. You can see the status of support of various environments in [Getting Started Guide](../docs/getting-started-guides/README.md).
+
+## Creating and configuring a Cluster
+
+Assuming that you have machines for your clusters ready, you need to install Kubernetes on them. For specific instructions you can consult one of existing Getting Started guides for various environments:
+* GKE [GKE page](https://cloud.google.com/container-engine),
+* Vagrant [Vagrant setup](../docs/getting-started-guides/vagrant.md),
+* GCE [GCE setup](../docs/getting-started-guides/gce.md),
+* CoreOS on Azure [CoreOS on Azure setup](../docs/getting-started-guides/coreos/azure/README.md),
+* CoreOS on AWS [CoreOS on AWS setup](../docs/getting-started-guides/coreos.md),
+* CoreOS on VMWare [CoreOS on VMWare setup](../docs/getting-started-guides/coreos.md).
+
+For more comprehensive and detailed description of cluster setup and configuration process see [From Scratch](../docs/getting-started-guides/scratch.md) guide.
+
+## Difference between user-oriented and ops-oriented tools
+
+Currently Kubernetes has two kinds of tools: developer oriented yet useful for operations people `kubectl` or `cluster/kubectl.sh` that use user specified configuration/contexts. `kubectl` is a stable version of `cluster/kubectl.sh` distributed with the `gcloud` stack.
+
+Cluster configuration/management tools stored in the `cluster` and `hack` folder, which are targeted to operations people, and only people managing clusters should use them. Those tools are mainly `bash` scripts which make managing single cluster easier, but they are currently not designed to allow easy management of multiple clusters from a single machine. If it's necessary it is possible to replicate existing e2e test cluster setup that consists mainly of `hack/ginkgo-e2e.sh` script and special test configs for specific cloud providers e.g. `cluster/gce/config-test.sh`. This means that in general scripts from `cluster` directory find a running cluster by themselves, and you don't need to explicitly specify the cluster you want to modify, as those tools assume that there's the cluster running.
```

Comment:
Actually, now everything in hack is for developers.  You can be an admin but not be a developer.  There will probably be many more admins than devs if we are doing it right.